app/controllers/search_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SearchController < ApplicationController
+  def index
+    ransack_params = params[:q]
+    # Hack to delete params can't be used by ransack
+    tag_search = ransack_params['tag_search'].split(',') if ransack_params.present? && ransack_params['tag_search'].present?
+    ransack_params.delete('tag_search') if ransack_params.present?
+
+    table = tag_search ? Slide.tagged_with(tag_search, any: true) : Slide
+    @q = table.search(ransack_params)
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name '@q' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->